packages/squiggle-lang/src/fr/dist.ts

```diff
@@ -144,6 +140,19 @@ function makeOneArgDist(
 }
 
 export const library: FRFunction[] = [
+  //We might want to later add all of the options to make() tht SampleSet has. For example, function() and list().
```

Comment:
// We might want to later add all of the options to make() that SampleSet has. For example, function() and list().